exo-go/src/service_helpers/index.go

```diff
@@ -0,0 +1,30 @@
+package serviceHelpers
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/Originate/exosphere/exo-go/src/types"
+	"github.com/Originate/exosphere/exo-go/src/util"
+)
+
+// VerifyServiceDoesNotExist forces the program to exit with status code 1
+// if the service serviceRole already exists in existingServices
+func VerifyServiceDoesNotExist(serviceRole string, existingServices []string) {
+	if util.DoesStringArrayContain(existingServices, serviceRole) {
+		fmt.Printf(`Service %v already exists in this application`, serviceRole)
+		os.Exit(1)
```

Comment:
Lets not exit from this inside this method but just return an error or not. Lets leave just the commands to exit when needed